Automattic/jetpack
# Contact Form: Fix Form placeholder background color

Fixes #27473

#### Changes proposed in this Pull Request:
* Changes the Form placeholder background color to white to prevent weird styling on colored backgrounds

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Create a post and add a Form block
* Check if the entire Form placeholder has a white background
